doc/user/content/sql/create-materialized-view.md

```diff
@@ -1,78 +1,92 @@
-query in memory. For more information, see [Key Concepts: Materialized views](/overview/key-concepts/#materialized-views).
+A materialized view specifies a [cluster](/overview/key-concepts/#clusters) that
+is tasked with maintaining its results up-to-date, but **can be referenced in
+any cluster**. This allows you to effectively decouple the computational
+resources used for view maintenance from the resources used for query serving.
 
 ## Syntax
 
 {{< diagram "create-materialized-view.svg" >}}
 
 Field | Use
 ------|-----
-**OR REPLACE** | If a view exists with the same name, replace it with the view defined in this statement. You cannot replace views that other views depend on, nor can you replace a non-view object with a view.
-**IF NOT EXISTS** | If specified, _do not_ generate an error if a view of the same name already exists. <br/><br/>If _not_ specified, throw an error if a view of the same name already exists. _(Default)_
-_view&lowbar;name_ | A name for the view.
+**OR REPLACE** | If a materialized view exists with the same name, replace it with the view defined in this statement. You cannot replace views that other views depend on, nor can you replace a non-view object with a view.
+**IF NOT EXISTS** | If specified, _do not_ generate an error if a materialized view of the same name already exists. <br/><br/>If _not_ specified, throw an error if a view of the same name already exists. _(Default)_
+_view&lowbar;name_ | A name for the materialized view.
 **(** _col_ident_... **)** | Rename the `SELECT` statement's columns to the list of identifiers, both of which must be the same length. Note that this is required for statements that return multiple columns with the same identifier.
-_cluster&lowbar;name_ | The cluster to maintain this materialized view. If not provided, uses the session’s cluster variable.
-_select&lowbar;stmt_ | The [`SELECT` statement](../select) whose output you want to materialize and maintain.
+_cluster&lowbar;name_ | The cluster to maintain this materialized view. If not specified, defaults to the active cluster.
+_select&lowbar;stmt_ | The [`SELECT` statement](../select) whose results you want to maintain incrementally updated.
 
 ## Details
 
-### Memory
+### Usage patterns
+
+Maintaining a materialized view in durable storage has resource and latency
+costs that should be carefully considered depending on the main usage of the
+view. It's a good idea to create a materialized view if:
 
-Views are maintained in memory. Because of this, one must be sure that all
-intermediate stages of the query, as well as its result set can fit in the
-memory of a single machine, while also understanding the rate at which the
-query's result set will grow.
+* The results need to be available across clusters;
+* View maintenance and query serving would benefit from being scaled
+  independently;
+* The final consumer of the view is a [subscription](../tail).
```

Comment:
@sjwiesman, I was following your lead in [this comment](https://materializeinc.slack.com/archives/C0246GEHL8N/p1659995132907849?thread_ts=1659983320.385319&cid=C0246GEHL8N), but didn't fully grasp what the advantage would be for sinks; if you create a sink on a materialized view, don't we just create a new storage collection (as opposed to reading out of the existing collection for the materialized view)?

Reply:
With sinks in platform, you'll _have_ to create a materialized view, unless we get back to https://www.notion.so/materialize/Internal-storage-sink-syntax-017251c532df465fb3a3ec7c344df54b#7de5b1eee5c54281bbe287f874341cce.